langchain4j-jlama/src/main/java/dev/langchain4j/model/jlama/JlamaChatLanguageModel.java

```diff
@@ -0,0 +1,85 @@
+package dev.langchain4j.model.jlama;
+
+import com.github.tjake.jlama.model.AbstractModel;
+import com.github.tjake.jlama.model.functions.Generator;
+import com.github.tjake.jlama.safetensors.tokenizer.PromptSupport;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.internal.RetryUtils;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.jlama.spi.JlamaChatLanguageModelBuilderFactory;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.output.TokenUsage;
+import lombok.Builder;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static dev.langchain4j.model.jlama.JlamaLanguageModel.toFinishReason;
+import static dev.langchain4j.spi.ServiceHelper.loadFactories;
+
+public class JlamaChatLanguageModel implements ChatLanguageModel {
```

Comment:
```suggestion
public class JlamaChatModel implements ChatLanguageModel {
```
To follow the same naming convention as other model providers